aep/general/0101/aep.md.j2

```diff
@@ -0,0 +1,11 @@
+# OpenAPI
+
+This AEP describes the usage of OpenAPI within the AEPs and AEP-compliant APIs.
+
+## Guidance
+
+### OAS version
+
+AEP-compliant APIs **should** document their HTTP APIs using
+[version 3.1](https://spec.openapis.org/oas/v3.1.1.html) of the OpenAPI
+Specification (OAS).
```

Comment:
We might want to add recommendations for adopting new versions, like moving to new minor versions within 6 months of their release, and moving to new major releases within 1 year.

Reply:
I think we could add that, but note that the tooling may not all update within 6 months / 1 year. I'm not sure we could make that commitment with the amount of dev time we have on the project.

Maybe keep a clause about continuing to expose older versions so tooling can still use it? Although I guess the worst case scenario is clients won't support features in newer versions of the OAS spec, which would be fine.

Reply:
done! PTAL.